src/i18n/lang/fr.json

```diff
@@ -208,6 +208,7 @@
 
   "FolderSelector": {
     "selectedFolder": "Dossier sélectionné",
+    "emptyFolderPlaceholder": "Dossier vide",
```

Comment:
Can you also add the English translation?

Reply:
@NicolasRichel what should be the process to ask for a PO Editor translation?

Reply:
I added the english version in en.json file ! Actually, this text is already translated in each language, I can update them manually ! as you want

Reply:
Is they already exist, you can copy them!

Reply:
Currently, the process is to add **fr** and **en** (if possible) translations to i18n files (*fr.json* & *en.json*) as we develop features. Then **manually** import them into POEditor when they are merged to **develop**.
Once french translations are available on POEditor, ask Gab or Lou to order translations in other languages.
When new translations are available, export them **manually** to the codebase.

The **import** / **export** operation should be automated in near future.

So, if you added translations to *fr.json* and *en.json*, that's fine for now. :)

Reply:
understood ! thank you explaining me the process @NicolasRichel